components/frontend/homePage/Footer.tsx

```diff
@@ -136,63 +119,49 @@ const Footer = () => {
           {/* Content for about links */}
           <Box>
             <Typography
-              variant="body2"
+              variant="h4"
+              fontWeight={700}
               sx={{
-                color: '#FFFFFF',
-                fontFamily: "Inter",
-                fontSize: "16px",
-                lineHeight: "normal",
-                fontStyle: "normal",
-                fontWeight: "700",
-                letterSpacing: "-0.232px",
+                "@media (max-width:600px)": {
+                  fontWeight: "bold",
+                },
               }}
             >
               {footerContent.footerSectionText2}
             </Typography>
             <Typography
-              variant="body2"
-              className={montserrat.className}
-              sx={{
-                color: '#FFFFFF',
-                fontFamily: "Montserrat, sans-serif",
-                fontSize: "14px",
-                lineHeight: "28px",
-                fontStyle: "normal",
-                fontWeight: "400",
-                marginTop: "15px",
-              }}
+              variant="h5"
+              fontFamily={"Mulish"}
+              color={"#FFFFFF"}
+              fontWeight={200}
             >
-             <Link href="/contact-us" style={{ color: "white", textDecoration:"none"}}> {footerContent.customText1}</Link>
+              <Link
+                href="/contact-us"
+                style={{ color: "white", textDecoration: "none" }}
+              >
+                {" "}
+                {footerContent.customText1}
+              </Link>
             </Typography>
             <Typography
-              variant="body2"
-              className={montserrat.className}
-              sx={{
-                color: '#FFFFFF',
-                fontFamily: "Montserrat, sans-serif",
-                fontSize: "14px",
-                lineHeight: "28px",
-                fontStyle: "normal",
-                fontWeight: "400",
-                marginTop: "5px",
-              }}
+              variant="h5"
+              fontFamily={"Mulish"}
+              color={"#FFFFFF"}
```

Comment:
Why have you applied {} here? I don't think it's necessary as you are passing a static value here.